pkg/minishift/config/instanceconfig.go

```diff
@@ -27,6 +27,7 @@ var InstanceConfig *InstanceConfigType
 type InstanceConfigType struct {
 	FilePath string `json:"-"`
 	OcPath   string
+	Register bool
```

Comment:
It depends whether you see this as an instruction/action for the system, aka 'minishift register this instance' or whether you describe a state, "this instance is registered".  I tend to think in the latter terms in this context.